spec/features/audit/lists/list_content_items_spec.rb

```diff
@@ -32,6 +32,17 @@
     expect(page).to have_text("2 items")
   end
 
+  scenario "start audit button goes to first audit item" do
```

Comment:
What about adding a test that checks the non-existence of the button when we have no content items?